Doc/c-api/dict.rst

```diff
@@ -173,6 +173,16 @@ Dictionary Objects
 
    .. versionadded:: 3.4
 
+
+.. c:function:: PyObject* PyDict_Pop(PyObject *p, PyObject *key, PyObject *defaultobj)
```

Comment:
I would prefer to return -1 on error, 0 if the key doesn't exist, 1 if the key exists.

Reply:
How would the caller know the popped _value_ then?